FAQInstall: add what happens wrong macOS package (#1264)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-Installation.md

@[youtube](_Zmc54XYzBA)
:::

:::details
### What happens when I install the wrong package for macOS?

> Wasabi will not start or crash at startup.

Reviewer: Nitpicking
```suggestion
Wasabi won't start or it crashes on startup.
```

Reviewer 2: This changes the meaning.

Reviewer: That was the point. I had to read the original few times before I understood it. This is supposed to be clarification.

Reviewer 2: I guess he meant it will not start and it will not crash. @MarnixCroes ?

Author: afaik what can happen is:
-Wasabi doesn't start
or
-Wasabi crashes a couple seconds after startup

> This will not directly harm your computer.

Reviewer 2: IMO we don't need that, and the way it is put seems like it can harm their computer indirectly.

> This will not directly harm your computer.

Author: ok, I wasn't sure.
I thought a newbie user: "when I download for the wrong processor this might harm my computer" so they're afraid to install either version.

can be removed though
